<html><head>
    <title>Card prototype</title>
<!--     <link href='https://fonts.googleapis.com/css?family=Roboto:400,700,300&subset=latin,greek,latin-ext' rel='stylesheet' type='text/css'> -->
    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:300,400,700&amp;subset=latin,greek,latin-ext" rel="stylesheet">
    <style>
    /* cyrillic-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/0eC6fl06luXEYWpBSJvXCBJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0460-052F, U+20B4, U+2DE0-2DFF, U+A640-A69F;
}
/* cyrillic */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/Fl4y0QdOxyyTHEGMXX8kcRJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
}
/* greek-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/-L14Jk06m6pUHB-5mXQQnRJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+1F00-1FFF;
}
/* greek */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/I3S1wsgSg9YCurV6PUkTORJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0370-03FF;
}
/* vietnamese */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/NYDWBdD4gIq26G5XYbHsFBJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0102-0103, U+1EA0-1EF9, U+20AB;
}
/* latin-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/Pru33qjShpZSmG3z6VYwnRJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0100-024F, U+1E00-1EFF, U+20A0-20AB, U+20AD-20CF, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url(https://fonts.gstatic.com/s/roboto/v15/Hgo13k-tfSpn0qi1SFdUfVtXRa8TVwTICgirnJhmVJw.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
}
/* cyrillic-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/ek4gzZ-GeXAPcSbHtCeQI_esZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+0460-052F, U+20B4, U+2DE0-2DFF, U+A640-A69F;
}
/* cyrillic */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/mErvLBYg_cXG3rLvUsKT_fesZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
}
/* greek-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/-2n2p-_Y08sg57CNWQfKNvesZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+1F00-1FFF;
}
/* greek */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/u0TOpm082MNkS5K0Q4rhqvesZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+0370-03FF;
}
/* vietnamese */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/NdF9MtnOpLzo-noMoG0miPesZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+0102-0103, U+1EA0-1EF9, U+20AB;
}
/* latin-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/Fcx7Wwv8OzT71A3E1XOAjvesZW2xOQ-xsNqO47m55DA.woff2) format('woff2');
  unicode-range: U+0100-024F, U+1E00-1EFF, U+20A0-20AB, U+20AD-20CF, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url(https://fonts.gstatic.com/s/roboto/v15/CWB0XYA8bzo0kSThX0UTuA.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
}
/* cyrillic-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/77FXFjRbGzN4aCrSFhlh3hJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0460-052F, U+20B4, U+2DE0-2DFF, U+A640-A69F;
}
/* cyrillic */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/isZ-wbCXNKAbnjo6_TwHThJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
}
/* greek-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/UX6i4JxQDm3fVTc1CPuwqhJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+1F00-1FFF;
}
/* greek */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/jSN2CGVDbcVyCnfJfjSdfBJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0370-03FF;
}
/* vietnamese */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/PwZc-YbIL414wB9rB1IAPRJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0102-0103, U+1EA0-1EF9, U+20AB;
}
/* latin-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/97uahxiqZRoncBaCEI3aWxJtnKITppOI_IvcXXDNrsc.woff2) format('woff2');
  unicode-range: U+0100-024F, U+1E00-1EFF, U+20A0-20AB, U+20AD-20CF, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url(https://fonts.gstatic.com/s/roboto/v15/d-6IYplOFocCacKzxwXSOFtXRa8TVwTICgirnJhmVJw.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
}
    </style>
    <style>

article, article ~ div {
  border:1px solid black;
  margin:35px;
  display:inline-block;
  background:#e8e8e8;
  float:left;
}

.card {
  position: relative;
  padding:16px;
  width:300px;
  
  border-bottom:1px solid rgba(0,0,0,.12);
  overflow: hidden;
    
  background: rgb(255, 255, 255);
  color: rgba(0,0,0,.87);
  font-family: Roboto;
  
  -webkit-transition: max-height 0.5s ease;
  -moz-transition: max-height 0.5s ease;
  transition: max-height 0.5s ease;
}

.card:not(:focus) {
  max-height: 166px;
}

.card:focus {
  max-height: 2000px;
  padding:8px;
  margin:8px;
  
  border:none;
  outline:none;
  border-radius: 2px;
  
  background: rgb(250, 250, 250);
  box-shadow: 0 2px 2px 0 rgba(0,0,0,0.14),0 3px 1px -2px rgba(0,0,0,0.2),0 1px 5px 0 rgba(0,0,0,0.12);
}

.card       .show { display:inline; }
.card       .hide { display:none  ; }
.card:focus .show { display:none  ; }
.card:focus .hide { display:inline; }

.card header {
  overflow:hidden;
  margin:8px 96px 0 0;
  height:93px;
}

.card h1 {
  font-weight:normal;
  max-height:2.33333333333em;
  font-size:24px;
  overflow:hidden;
  margin:0;
}

.card section p {
  margin:5px 0 0 0;
  font-size:14px;
  overflow:hidden;
}

.card section p span,
.card section p a {
  color:rgba(0,0,0,.54);
}

.card main {
  margin:8px 0 0 0;
/*   margin:0 -8px -8px -8px; */
}

.card > img {
  position:absolute;
  top:16px;
  right:16px;
  width:80px;
  height:80px;
  object-fit:contain;
}

@font-face {
  font-family: 'Material Icons';
  font-style: normal;
  font-weight: 400;
  src: local('Material Icons'), local('MaterialIcons-Regular'), url(https://fonts.gstatic.com/s/materialicons/v17/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
}

.material-icons,
.material-icons-before:before,
.material-icons-after:after {
  font-family: 'Material Icons';
  font-weight: normal;
  font-style: normal;
  font-size: 24px;
  line-height: 1;
  letter-spacing: normal;
  text-transform: none;
  display: inline-block;
  white-space: nowrap;
  word-wrap: normal;
  direction: ltr;
  -webkit-font-feature-settings: 'liga';
  -webkit-font-smoothing: antialiased;
}


span.count:before {
  content:' - ';
}

    </style>
  </head>
  <body>
  
<article>
  <div class="card" tabindex="-1">
    <header>
      <h1>Sequencing of the needle transcriptome from Norway spruce (Picea abies Karst L.) reveals lower substitution rates, but similar selective constraints in gymnosperms and angiosperms.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1471-2164-13-589">10.1186/1471-2164-13-589</a></span>, <span>BMC genomics</span></p>
      </section>
    </header>
    <img src="undefined">
    <main> BACKGROUND: A detailed knowledge about spatial and temporal gene expression is important for understanding both the function of genes and their evolution. For the vast majority of species, transcriptomes are still largely uncharacterized and even in those where substantial information is available it is often in the form of partially sequenced transcriptomes. With the development of next generation sequencing, a single experiment can now simultaneously identify the transcribed part of a species genome and estimate levels of gene expression. RESULTS: mRNA from actively growing needles of Norway spruce (Picea abies) was sequenced using next generation sequencing technology. In total, close to 70 million fragments with a length of 76 bp were sequenced resulting in 5 Gbp of raw data. A de novo assembly of these reads, together with publicly available expressed sequence tag (EST) data from Norway spruce, was used to create a reference transcriptome. Of the 38,419 PUTs (putative unique transcripts) longer than 150 bp in this reference assembly, 83.5% show similarity to ESTs from other spruce species and of the remaining PUTs, 3,704 show similarity to protein sequences from other plant species, leaving 4,167 PUTs with limited similarity to currently available plant proteins. By predicting coding frames and comparing not only the Norway spruce PUTs, but also PUTs from the close relatives Picea glauca and Picea sitchensis to both Pinus taeda and Taxus mairei, we obtained estimates of synonymous and non-synonymous divergence among conifer species. In addition, we detected close to 15,000 SNPs of high quality and estimated gene expression differences between samples collected under dark and light conditions. CONCLUSIONS: Our study yielded a large number of single nucleotide polymorphisms as well as estimates of gene expression on transcriptome scale. In agreement with a recent study we find that the synonymous substitution rate per year (0.6 × 10-09 and 1.1 × 10-09) is an order of magnitude smaller than values reported for angiosperm herbs. However, if one takes generation time into account, most of this difference disappears. The estimates of the dN/dS ratio (non-synonymous over synonymous divergence) reported here are in general much lower than 1 and only a few genes showed a ratio larger than 1.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A genome-wide association study identifies genomic regions for virulence in the non-model organism Heterobasidion annosum s.s.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0053525">10.1371/journal.pone.0053525</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>The dense single nucleotide polymorphisms (SNP) panels needed for genome wide association (GWA) studies have hitherto been expensive to establish and use on non-model organisms. To overcome this, we used a next generation sequencing approach to both establish SNPs and to determine genotypes. We conducted a GWA study on a fungal species, analysing the virulence of Heterobasidion annosum s.s., a necrotrophic pathogen, on its hosts Picea abies and Pinus sylvestris. From a set of 33,018 single nucleotide polymorphisms (SNP) in 23 haploid isolates, twelve SNP markers distributed on seven contigs were associated with virulence (P&lt;0.0001). Four of the contigs harbour known virulence genes from other fungal pathogens and the remaining three harbour novel candidate genes. Two contigs link closely to virulence regions recognized previously by QTL mapping in the congeneric hybrid H. irregulare × H. occidentale. Our study demonstrates the efficiency of GWA studies for dissecting important complex traits of small populations of non-model haploid organisms with small genomes.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Potential for evolutionary responses to climate change - evidence from tree populations.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1111/gcb.12181">10.1111/gcb.12181</a></span>, <span>Global change biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Evolutionary responses are required for tree populations to be able to track climate change. Results of 250 years of common garden experiments show that most forest trees have evolved local adaptation, as evidenced by the adaptive differentiation of populations in quantitative traits, reflecting environmental conditions of population origins. On the basis of the patterns of quantitative variation for 19 adaptation-related traits studied in 59 tree species (mostly temperate and boreal species from the Northern hemisphere), we found that genetic differentiation between populations and clinal variation along environmental gradients were very common (respectively, 90% and 78% of cases). Thus, responding to climate change will likely require that the quantitative traits of populations again match their environments. We examine what kind of information is needed for evaluating the potential to respond, and what information is already available. We review the genetic models related to selection responses, and what is known currently about the genetic basis of the traits. We address special problems to be found at the range margins, and highlight the need for more modeling to understand specific issues at southern and northern margins. We need new common garden experiments for less known species. For extensively studied species, new experiments are needed outside the current ranges. Improving genomic information will allow better prediction of responses. Competitive and other interactions within species and interactions between species deserve more consideration. Despite the long generation times, the strong background in quantitative genetics and growing genomic resources make forest trees useful species for climate change research. The greatest adaptive response is expected when populations are large, have high genetic variability, selection is strong, and there is ecological opportunity for establishment of better adapted genotypes.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Aluminum exclusion and aluminum tolerance in woody plants.</h1>
      <section>
        <p><span><a href="https://doi.org/10.3389/fpls.2013.00172">10.3389/fpls.2013.00172</a></span>, <span>Frontiers in plant science</span></p>
      </section>
    </header>
    <img src="https://api-journal.frontiersin.org/Areas/Header/Content/Images/thin-header-logo.png">
    <main>The aluminum (Al) cation Al(3) (+) is highly rhizotoxic and is a major stress factor to plants on acid soils, which cover large areas of tropical and boreal regions. Many woody plant species are native to acid soils and are well adapted to high Al(3) (+) conditions. In tropical regions, both woody Al accumulator and non-Al accumulator plants occur, whereas in boreal regions woody plants are non-Al accumulators. The mechanisms of these adaptations can be divided into those that facilitate the exclusion of Al(3) (+) from root cells (exclusion mechanisms) and those that enable plants to tolerate Al(3) (+) once it has entered the root and shoot symplast (internal tolerance mechanisms). The biochemical and molecular basis of these mechanisms have been intensively studied in several crop plants and the model plant Arabidopsis. In this review, we examine the current understanding of Al(3) (+) exclusion and tolerance mechanisms from woody plants. In addition, we discuss the ecology of woody non-Al accumulator and Al accumulator plants, and present examples of Al(3) (+) adaptations in woody plant populations. This paper complements previous reviews focusing on crop plants and provides insights into evolutionary processes operating in plant communities that are widespread on acid soils.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Analysis of the WUSCHEL-RELATED HOMEOBOX gene family in the conifer picea abies reveals extensive conservation as well as dynamic patterns.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1471-2229-13-89">10.1186/1471-2229-13-89</a></span>, <span>BMC plant biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main> BACKGROUND: Members of the WUSCHEL-RELATED HOMEOBOX (WOX) gene family have important functions during all stages of plant development and have been implicated in the development of morphological novelties during evolution. Most studies have examined the function of these genes in angiosperms and very little is known from other plant species. RESULTS: In this study we examine the presence and expression of WOX genes in the conifer Picea abies. We have cloned 11 WOX genes from both mRNA and genomic DNA and examined their phylogenetic relationship to WOX genes from other species as well as their expression during somatic embryogenesis and in adult tissues. CONCLUSIONS: Our study shows that all major radiations within the WOX gene family took place before the angiosperm-gymnosperm split and that there has been a recent expansion within the intermediate clade in the Pinaceae family. Furthermore, we show that the genes from the intermediate clade are preferentially expressed during embryo development in Picea abies. Our data also indicates that there are clear orthologs of both WUS and WOX5 present in the P. abies genome.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Open access to tree genomes: the path to a better forest.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/gb-2013-14-6-120">10.1186/gb-2013-14-6-120</a></span>, <span>Genome biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>An open-access culture and a well-developed comparative-genomics infrastructure must be developed in forest trees to derive the full potential of genome sequencing in this diverse group of plants that are the dominant species in much of the earth's terrestrial ecosystems.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A conifer genome spruces up plant phylogenomics.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/gb-2013-14-6-122">10.1186/gb-2013-14-6-122</a></span>, <span>Genome biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The Norway spruce genome provides key insights into the evolution of plant genomes, leading to testable new hypotheses about conifer, gymnosperm, and vascular plant evolution.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Forest structure, stand composition, and climate-growth response in montane forests of Jiuzhaigou National Nature Reserve, China.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0071559">10.1371/journal.pone.0071559</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Montane forests of western China provide an opportunity to establish baseline studies for climate change. The region is being impacted by climate change, air pollution, and significant human impacts from tourism. We analyzed forest stand structure and climate-growth relationships from Jiuzhaigou National Nature Reserve in northwestern Sichuan province, along the eastern edge of the Tibetan plateau. We conducted a survey to characterize forest stand diversity and structure in plots occurring between 2050 and 3350 m in elevation. We also evaluated seedling and sapling recruitment and tree-ring data from four conifer species to assess: 1) whether the forest appears in transition toward increased hardwood composition; 2) if conifers appear stressed by recent climate change relative to hardwoods; and 3) how growth of four dominant species responds to recent climate. Our study is complicated by clear evidence of 20(th) century timber extraction. Focusing on regions lacking evidence of logging, we found a diverse suite of conifers (Pinus, Abies, Juniperus, Picea, and Larix) strongly dominate the forest overstory. We found population size structures for most conifer tree species to be consistent with self-replacement and not providing evidence of shifting composition toward hardwoods. Climate-growth analyses indicate increased growth with cool temperatures in summer and fall. Warmer temperatures during the growing season could negatively impact conifer growth, indicating possible seasonal climate water deficit as a constraint on growth. In contrast, however, we found little relationship to seasonal precipitation. Projected warming does not yet have a discernible signal on trends in tree growth rates, but slower growth with warmer growing season climates suggests reduced potential future forest growth.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Tree migration-rates: narrowing the gap between inferred post-glacial rates and projected rates.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0071797">10.1371/journal.pone.0071797</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Faster-than-expected post-glacial migration rates of trees have puzzled ecologists for a long time. In Europe, post-glacial migration is assumed to have started from the three southern European peninsulas (southern refugia), where large areas remained free of permafrost and ice at the peak of the last glaciation. However, increasing palaeobotanical evidence for the presence of isolated tree populations in more northerly microrefugia has started to change this perception. Here we use the Northern Eurasian Plant Macrofossil Database and palaeoecological literature to show that post-glacial migration rates for trees may have been substantially lower (60-260 m yr(-1)) than those estimated by assuming migration from southern refugia only (115-550 m yr(-1)), and that early-successional trees migrated faster than mid- and late-successional trees. Post-glacial migration rates are in good agreement with those recently projected for the future with a population dynamical forest succession and dispersal model, mainly for early-successional trees and under optimal conditions. Although migration estimates presented here may be conservative because of our assumption of uniform dispersal, tree migration-rates clearly need reconsideration. We suggest that small outlier populations may be a key factor in understanding past migration rates and in predicting potential future range-shifts. The importance of outlier populations in the past may have an analogy in the future, as many tree species have been planted beyond their natural ranges, with a more beneficial microclimate than their regional surroundings. Therefore, climate-change-induced range-shifts in the future might well be influenced by such microrefugia.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Insights into the loblolly pine genome: characterization of BAC and fosmid sequences.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0072439">10.1371/journal.pone.0072439</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Despite their prevalence and importance, the genome sequences of loblolly pine, Norway spruce, and white spruce, three ecologically and economically important conifer species, are just becoming available to the research community. Following the completion of these large assemblies, annotation efforts will be undertaken to characterize the reference sequences. Accurate annotation of these ancient genomes would be aided by a comprehensive repeat library; however, few studies have generated enough sequence to fully evaluate and catalog their non-genic content. In this paper, two sets of loblolly pine genomic sequence, 103 previously assembled BACs and 90,954 newly sequenced and assembled fosmid scaffolds, were analyzed. Together, this sequence represents 280 Mbp (roughly 1% of the loblolly pine genome) and one of the most comprehensive studies of repetitive elements and genes in a gymnosperm species. A combination of homology and de novo methodologies were applied to identify both conserved and novel repeats. Similarity analysis estimated a repetitive content of 27% that included both full and partial elements. When combined with the de novo investigation, the estimate increased to almost 86%. Over 60% of the repetitive sequence consists of full or partial LTR (long terminal repeat) retrotransposons. Through de novo approaches, 6,270 novel, full-length transposable element families and 9,415 sub-families were identified. Among those 6,270 families, 82% were annotated as single-copy. Several of the novel, high-copy families are described here, with the largest, PtPiedmont, comprising 133 full-length copies. In addition to repeats, analysis of the coding region reported 23 full-length eukaryotic orthologous proteins (KOGS) and another 29 novel or orthologous genes. These discoveries, along with other genomic resources, will be used to annotate conifer genomes and address long-standing questions about gymnosperm evolution.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Metabolomics reveals the origins of antimicrobial plant resins collected by honey bees.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0077512">10.1371/journal.pone.0077512</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>The deposition of antimicrobial plant resins in honey bee, Apis mellifera, nests has important physiological benefits. Resin foraging is difficult to approach experimentally because resin composition is highly variable among and between plant families, the environmental and plant-genotypic effects on resins are unknown, and resin foragers are relatively rare and often forage in unobservable tree canopies. Subsequently, little is known about the botanical origins of resins in many regions or the benefits of specific resins to bees. We used metabolomic methods as a type of environmental forensics to track individual resin forager behavior through comparisons of global resin metabolite patterns. The resin from the corbiculae of a single bee was sufficient to identify that resin's botanical source without prior knowledge of resin composition. Bees from our apiary discriminately foraged for resin from eastern cottonwood (Populus deltoides), and balsam poplar (P. balsamifera) among many available, even closely related, resinous plants. Cottonwood and balsam poplar resin composition did not show significant seasonal or regional changes in composition. Metabolomic analysis of resin from 6 North American Populus spp. and 5 hybrids revealed peaks characteristic to taxonomic nodes within Populus, while antimicrobial analysis revealed that resin from different species varied in inhibition of the bee bacterial pathogen, Paenibacillus larvae. We conclude that honey bees make discrete choices among many resinous plant species, even among closely related species. Bees also maintained fidelity to a single source during a foraging trip. Furthermore, the differential inhibition of P. larvae by Populus spp., thought to be preferential for resin collection in temperate regions, suggests that resins from closely related plant species many have different benefits to bees.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Molecular characterization and development of real-time PCR assay for pine-wood nematode Bursaphelenchus xylophilus (Nematoda: Parasitaphelenchidae).</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0078804">10.1371/journal.pone.0078804</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Bursaphelenchus xylophilus, the pine-wood nematode (PWN), is the causal agent of pine wilt disease, one of the most damaging emerging pest problems to forests around the world. It is native to North America where it causes relatively minor damage to native conifers but is labeled an EPPO-A-2 pest and a quarantine nematode for many countries outside of the United States because of its potential for destruction to their native conifers. Exports of wood logs and commodities involving softwood packaging materials now require a lab test for the presence/absence of this regulated nematode species. We characterized the DNA sequences on the ribosomal DNA small subunit, large subunit D2/D3, internal transcribed spacer (ITS) and mitochondrial DNA cytochrome oxidase subunit one on the aphelenchid species and described the development of a real-time-PCR method for rapid and accurate identification of PWN targeting the ITS-1. A total of 97 nematode populations were used to evaluate the specificity and sensitivity of this assay, including 45 populations of B. xylophilus and 36 populations of 21 other species of Bursaphelenchus which belong to the abietinus, cocophilus, eggersi, fungivorus, hofmanni, kevini, leoni, sexdentati, and xylophilus groups and one unassigned group from a total of 13 groups in the genus Bursaphelenchus; 15 populations of Aphelenchoides besseyi, A. fragariae, Aphelenchoides species and Aphelenchus avenae; and one population of mixed nematode species from a soil sample. This assay proved to be specific to B. xylophilus only and was sensitive to a single nematode specimen regardless of the life stages present. This approach provides rapid species identification necessary to comply with the zero-tolerance export regulations.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>The transcriptomics of secondary growth and wood formation in conifers.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1155/2013/974324">10.1155/2013/974324</a></span>, <span>Molecular biology international</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>In the last years, forestry scientists have adapted genomics and next-generation sequencing (NGS) technologies to the search for candidate genes related to the transcriptomics of secondary growth and wood formation in several tree species. Gymnosperms, in particular, the conifers, are ecologically and economically important, namely, for the production of wood and other forestry end products. Until very recently, no whole genome sequencing of a conifer genome was available. Due to the gradual improvement of the NGS technologies and inherent bioinformatics tools, two draft assemblies of the whole genomes sequence of Picea abies and Picea glauca arose in the current year. These draft genome assemblies will bring new insights about the structure, content, and evolution of the conifer genomes. Furthermore, new directions in the forestry, breeding and research of conifers will be discussed in the following. The identification of genes associated with the xylem transcriptome and the knowledge of their regulatory mechanisms will provide less time-consuming breeding cycles and a high accuracy for the selection of traits related to wood production and quality. <span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Phylogenomics and coalescent analyses resolve extant seed plant relationships.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0080870">10.1371/journal.pone.0080870</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>The extant seed plants include more than 260,000 species that belong to five main lineages: angiosperms, conifers, cycads, Ginkgo, and gnetophytes. Despite tremendous effort using molecular data, phylogenetic relationships among these five lineages remain uncertain. Here, we provide the first broad coalescent-based species tree estimation of seed plants using genome-scale nuclear and plastid data By incorporating 305 nuclear genes and 47 plastid genes from 14 species, we identify that i) extant gymnosperms (i.e., conifers, cycads, Ginkgo, and gnetophytes) are monophyletic, ii) gnetophytes exhibit discordant placements within conifers between their nuclear and plastid genomes, and iii) cycads plus Ginkgo form a clade that is sister to all remaining extant gymnosperms. We additionally observe that the placement of Ginkgo inferred from coalescent analyses is congruent across different nucleotide rate partitions. In contrast, the standard concatenation method produces strongly supported, but incongruent placements of Ginkgo between slow- and fast-evolving sites. Specifically, fast-evolving sites yield relationships in conflict with coalescent analyses. We hypothesize that this incongruence may be related to the way in which concatenation methods treat sites with elevated nucleotide substitution rates. More empirical and simulation investigations are needed to understand this potential weakness of concatenation methods.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Occurrence of species of the genus Pityophthorus Eichhoff (Coleoptera, Curculionidae, Scolytinae) in&nbsp;the&nbsp;province of Quebec, Canada.</h1>
      <section>
        <p><span><a href="https://doi.org/10.3897/zookeys.348.6029">10.3897/zookeys.348.6029</a></span>, <span>ZooKeys</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Twig beetles in the genus Pityophthorus Eichhoff, 1864 include more than 300 species worldwide, with maximum diversity in tropical and subtropical regions. To date, approximately 50 species of Pityophthorus have been recorded in Canada, and these species are associated mainly with coniferous trees. Since 1981, no comprehensive study on this difficult taxonomic group has been conducted in Quebec, Canada, most likely due to their limited significance as forest pests. Based on data gathered from five years of field sampling in conifer seed orchards and compiled from various entomological collections, the distribution of Pityophthorus species in Quebec is presented. Approximately 291 new localities were recorded for the Pityophthorus species. Five species-group taxa, namely Pityophthorus puberulus (LeConte, 1868), Pityophthorus pulchellus pulchellus Eichhoff, 1869, Pityophthorus pulicarius (Zimmermann, 1868), Pityophthorus nitidus Swaine, 1917,and Pityophthorus cariniceps LeConte&amp;Horn, 1876 were the most widespread. In contrast, Pityophthorus consimilis LeConte, 1878, Pityophthorus intextus Swaine, 1917, Pityophthorus dentifrons Blackman, 1922, Pityophthorus ramiperda Swaine, 1917, and Pityophthorus concavus Blackman, 1928 display a notably limited distribution. In addition, the first distribution records of Pityophthorus intextus and Pityophthorus biovalis Blackman, 1922 are furnished, and the subspecies Pityophthorus murrayanae murrayanae Blackman, 1922is reported from Quebec for the second time. Moreover, distribution maps are provided for all Pityophthorus species recorded in the province of Quebec. <span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Bark flammability as a fire-response trait for subalpine trees.</h1>
      <section>
        <p><span><a href="https://doi.org/10.3389/fpls.2013.00466">10.3389/fpls.2013.00466</a></span>, <span>Frontiers in plant science</span></p>
      </section>
    </header>
    <img src="https://api-journal.frontiersin.org/Areas/Header/Content/Images/thin-header-logo.png">
    <main>Relationships between the flammability properties of a given plant and its chances of survival after a fire still remain unknown. We hypothesize that the bark flammability of a tree reduces the potential for tree survival following surface fires, and that if tree resistance to fire is provided by a thick insulating bark, the latter must be few flammable. We test, on subalpine tree species, the relationship between the flammability of bark and its insulating ability, identifies the biological traits that determine bark flammability, and assesses their relative susceptibility to surface fires from their bark properties. The experimental set of burning properties was analyzed by Principal Component Analysis to assess the bark flammability. Bark insulating ability was expressed by the critical time to cambium kill computed from bark thickness. Log-linear regressions indicated that bark flammability varies with the bark thickness and the density of wood under bark and that the most flammable barks have poor insulating ability. Susceptibility to surface fires increases from gymnosperm to angiosperm subalpine trees. The co-dominant subalpine species Larix decidua (Mill.) and Pinus cembra (L.) exhibit large differences in both flammability and insulating ability of the bark that should partly explain their contrasted responses to fires in the past. <span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Is ecological speciation a major trend in aphids? Insights from a molecular phylogeny of the conifer-feeding genus Cinara.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1742-9994-10-56">10.1186/1742-9994-10-56</a></span>, <span>Frontiers in zoology</span></p>
      </section>
    </header>
    <img src="https://api-journal.frontiersin.org/Areas/Header/Content/Images/thin-header-logo.png">
    <main> INTRODUCTION: In the past decade ecological speciation has been recognized as having an important role in the diversification of plant-feeding insects. Aphids are host-specialised phytophagous insects that mate on their host plants and, as such, they are prone to experience reproductive isolation linked with host plant association that could ultimately lead to species formation. The generality of such a scenario remains to be tested through macroevolutionary studies. To explore the prevalence of host-driven speciation in the diversification of the aphid genus Cinara and to investigate alternative modes of speciation, we reconstructed a phylogeny of this genus based on mitochondrial, nuclear and Buchnera aphidicola DNA sequence fragments and applied a DNA-based method of species delimitation. Using a recent software (PhyloType), we explored evolutionary transitions in host-plant genera, feeding sites and geographic distributions in the diversification of Cinara and investigated how transitions in these characters have accompanied speciation events. RESULTS: The diversification of Cinara has been constrained by host fidelity to conifer genera sometimes followed by sequential colonization onto different host species and by feeding-site specialisation. Nevertheless, our analyses suggest that, at the most, only half of the speciation events were accompanied by ecological niche shifts. The contribution of geographical isolation in the speciation process is clearly apparent in the occurrence of species from two continents in the same clades in relatively terminal positions in our phylogeny. Furthermore, in agreement with predictions from scenarios in which geographic isolation accounts for speciation events, geographic overlap between species increased significantly with time elapsed since their separation. CONCLUSIONS: The history of Cinara offers a different perspective on the mode of speciation of aphids than that provided by classic models such as the pea aphid. In this genus of aphids, the role of climate and landscape history has probably been as important as host-plant specialisation in having shaped present-day diversity.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>500-year climate cycles stacking of recent centennial warming documented in an East Asian pollen record.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1038/srep03611">10.1038/srep03611</a></span>, <span>Scientific reports</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Here we presented a high-resolution 5350-year pollen record from a maar annually laminated lake in East Asia (EA). Pollen record reflected the dynamics of vertical vegetation zones and temperature change. Spectral analysis on pollen percentages/concentrations of Pinus and Quercus, and a temperature proxy, revealed ~500-year quasi-periodic cold-warm fluctuations during the past 5350 years. This ~500-year cyclic climate change occurred in EA during the mid-late Holocene and even the last 150 years dominated by anthropogenic forcing. It was almost in phase with a ~500-year periodic change in solar activity and Greenland temperature change, suggesting that ~500-year small variations in solar output played a prominent role in the mid-late Holocene climate dynamics in EA, linked to high latitude climate system. Its last warm phase might terminate in the next several decades to enter another ~250-year cool phase, and thus this future centennial cyclic temperature minimum could partially slow down man-made global warming.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Taxonomy and phylogeny of European Gymnopus subsection Levipedes (Basidiomycota, Omphalotaceae).</h1>
      <section>
        <p><span><a href="https://doi.org/10.3767/003158513x674043">10.3767/003158513x674043</a></span>, <span>Persoonia</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The systematic integrity of European Gymnopus subsect. Levipedes is verified based on anatomic-morphological characters with support from DNA sequences of ITS and translation elongation factor 1-alpha genes. Seven species (G. alpinus, G. aquosus, G. dryophilus - including var. lanipes, G. erythropus, G. fagiphilus, G. hybridus, and G. ocior) belonging to this subsection are included. We clarify the concepts of G. dryophilus and G. ocior, which were occasionally confused in older literature. Due to unavailability of previously selected neotype of G. dryophilus the substitute neotype specimen is selected. Gymnopus dryophilus var. lanipes is confirmed as a variety; no important differences from nominotypical variability were detected. All discriminative characters used for identification of these species are discussed in detail. An identification key is also provided.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Relationships within aphids Cinara (Cupressobium) (Hemiptera) based on mitochondrial and nuclear DNA sequences.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1007/s13353-013-0184-7">10.1007/s13353-013-0184-7</a></span>, <span>Journal of applied genetics</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The relationships between Cinara (Cupressobium) aphids inhabiting woody parts and leaves of conifers belonging to Cupressaceae have been studied using a mitochondrial gene (COI) and a nuclear gene (EF1-α). Based on the COI sequences, genetic distances between species ranged from 5.6&nbsp;% between Cinara (C.) tujafilina (del Guercio) and Cinara (C.) juniperi (De Geer) to 10.5&nbsp;% between C. (C.) tujafilina and Cinara (C.) mordvilkoi (Pašek). Genetic distances among EF1-α sequences were lower and showed from 0.1&nbsp;% between C. cupressi and C. juniperi to 2.3&nbsp;% between C. tujafilina and C. mordvilkoi. Molecular phylogenetic trees were constructed using the Bayesian inference (BI) phylogenetic analysis and maximum parsimony (MP) criterion. Phylogenetic trees obtained based on COI and EF1-α marker genes created two sister clades. Our results indicate that Cinara (Cupressobium) are a monophyletic group of aphids. Phylogenetic relationships amongst Cupressobium aphids do not result from the association with the host plant, but from the feeding site on the host plant or an ability to change the microhabitat on the plant. As closely related species inhabit similar microhabitats on different host plants, it suggests that the host switching is the main mode of speciation in this subgenus.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Network analysis reveals ecological links between N-fixing bacteria and wood-decaying fungi.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0088141">10.1371/journal.pone.0088141</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Nitrogen availability in dead wood is highly restricted and associations with N-fixing bacteria are thought to enable wood-decaying fungi to meet their nitrogen requirements for vegetative and generative growth. We assessed the diversity of nifH (dinitrogenase reductase) genes in dead wood of the common temperate tree species Fagus sylvatica and Picea abies from differently managed forest plots in Germany using molecular tools. By incorporating these genes into a large compilation of published nifH sequences and subsequent phylogenetic analyses of deduced proteins we verified the presence of diverse pools corresponding to functional nifH, almost all of which are new to science. The distribution of nifH genes strongly correlated with tree species and decay class, but not with forest management, while higher fungal fructification was correlated with decreasing nitrogen content of the dead wood and positively correlated with nifH diversity, especially during the intermediate stage of wood decay. Network analyses based on non-random species co-occurrence patterns revealed interactions among fungi and N-fixing bacteria in the dead wood and strongly indicate the occurrence of at least commensal relationships between these taxa.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Are temperate canopy spiders tree-species specific?</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0086571">10.1371/journal.pone.0086571</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Arboreal spiders in deciduous and coniferous trees were investigated on their distribution and diversity. Insecticidal knock-down was used to comprehensively sample spiders from 175 trees from 2001 to 2003 in the Białowieża forest and three remote forests in Poland. We identified 140 species from 9273 adult spiders. Spider communities were distinguished between deciduous and coniferous trees. The richest fauna was collected from Quercus where beta diversity was also highest. A tree-species-specific pattern was clearly observed for Alnus, Carpinus, Picea and Pinus trees and also for those tree species that were fogged in only four or three replicates, namely Betula and Populus. This hitherto unrecognised association was mainly due to the community composition of common species identified in a Dufrene-Legendre indicator species analysis. It was not caused by spatial or temporal autocorrelation. Explaining tree-species specificity for generalist predators like spiders is difficult and has to involve physical and ecological tree parameters like linkage with the abundance of prey species. However, neither did we find a consistent correlation of prey group abundances with spiders nor could differences in spider guild composition explain the observed pattern. Our results hint towards the importance of deterministic mechanisms structuring communities of generalist canopy spiders although the casual relationship is not yet understood.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Unique features of the loblolly pine (Pinus taeda L.) megagenome revealed through sequence annotation.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1534/genetics.113.159996">10.1534/genetics.113.159996</a></span>, <span>Genetics</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The largest genus in the conifer family Pinaceae is Pinus, with over 100 species. The size and complexity of their genomes (∼20-40 Gb, 2n = 24) have delayed the arrival of a well-annotated reference sequence. In this study, we present the annotation of the first whole-genome shotgun assembly of loblolly pine (Pinus taeda L.), which comprises 20.1 Gb of sequence. The MAKER-P annotation pipeline combined evidence-based alignments and ab initio predictions to generate 50,172 gene models, of which 15,653 are classified as high confidence. Clustering these gene models with 13 other plant species resulted in 20,646 gene families, of which 1554 are predicted to be unique to conifers. Among the conifer gene families, 159 are composed exclusively of loblolly pine members. The gene models for loblolly pine have the highest median and mean intron lengths of 24 fully sequenced plant genomes. Conifer genomes are full of repetitive DNA, with the most significant contributions from long-terminal-repeat retrotransposons. In depth analysis of the tandem and interspersed repetitive content yielded a combined estimate of 82%.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Extraction and Chromatographic Determination of Shikimic Acid in Chinese Conifer Needles with 1-Benzyl-3-methylimidazolium Bromide Ionic Liquid Aqueous Solutions.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1155/2014/256473">10.1155/2014/256473</a></span>, <span>Journal of analytical methods in chemistry</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>An ionic liquids-based ultrasound-assisted extraction (ILUAE) method was successfully developed for extracting shikimic acid from conifer needles. Eleven 1-alkyl-3-methylimidazolium ionic liquids with different cations and anions were investigated and 1-benzyl-3-methylimidazolium bromide solution was selected as the solvent. The conditions for ILUAE, including the ionic liquid concentration, ultrasound power, ultrasound time, and liquid-solid ratio, were optimized. The proposed method had good recovery (99.37%-100.11%) and reproducibility (RSD, n = 6; 3.6%). ILUAE was an efficient, rapid, and simple sample preparation technique that showed high reproducibility. Based on the results, a number of plant species, namely, Picea koraiensis, Picea meyeri, Pinus elliottii, and Pinus banksiana, were identified as among the best resources of shikimic acid.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Scaling relationships between leaf mass and total plant mass across Chinese forests.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0095938">10.1371/journal.pone.0095938</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Biomass partitioning is important for illustrating terrestrial ecosystem carbon flux. West, Brown and Enquist (WBE) model predicts that an optimal 3/4 allometric scaling of leaf mass and total biomass of individual plants will be applied in diverse communities. However, amount of scientific evidence suggests an involvement of some biological and environmental factors in interpreting the variation of scaling exponent observed in empirical studies. In this paper, biomass information of 1175 forested communities in China was collected and categorized into groups in terms of leaf form and function, as well as their locations to test whether the allocation pattern was conserved or variable with internal and/or environmental variations. Model Type II regression protocol was adopted to perform all the regressions. The results empirically showed that the slopes varied significantly across diverse forested biomes, between conifer and broadleaved forests, and between evergreen and deciduous forests. Based on the results, leaf form and function and their relations to environments play a significant role in the modification of the WBE model to explore more accurate laws in nature.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Patterns of nucleotide diversity at photoperiod related genes in Norway spruce [Picea abies (L.) Karst].</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0095306">10.1371/journal.pone.0095306</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>The ability of plants to track seasonal changes is largely dependent on genes assigned to the photoperiod pathway, and variation in those genes is thereby important for adaptation to local day length conditions. Extensive physiological data in several temperate conifer species suggest that populations are adapted to local light conditions, but data on the genes underlying this adaptation are more limited. Here we present nucleotide diversity data from 19 genes putatively involved in photoperiodic response in Norway spruce (Picea abies). Based on similarity to model plants the genes were grouped into three categories according to their presumed position in the photoperiod pathway: photoreceptors, circadian clock genes, and downstream targets. An HKA (Hudson, Kreitman and Aquade) test showed a significant excess of diversity at photoreceptor genes, but no departure from neutrality at circadian genes and downstream targets. Departures from neutrality were also tested with Tajima's D and Fay and Wu's H statistics under three demographic scenarios: the standard neutral model, a population expansion model, and a more complex population split model. Only one gene, the circadian clock gene PaPRR3 with a highly positive Tajima's D value, deviates significantly from all tested demographic scenarios. As the PaPRR3 gene harbours multiple non-synonymous variants it appears as an excellent candidate gene for control of photoperiod response in Norway spruce.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Composition and structure of Pinus koraiensis mixed forest respond to spatial climatic changes.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0097192">10.1371/journal.pone.0097192</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>BACKGROUND: Although some studies have indicated that climate changes can affect Pinus koraiensis mixed forest, the responses of composition and structure of Pinus koraiensis mixed forests to climatic changes are unknown and the key climatic factors controlling the composition and structure of Pinus koraiensis mixed forest are uncertain. METHODOLOGY/PRINCIPAL FINDINGS: Field survey was conducted in the natural Pinus koraiensis mixed forests along a latitudinal gradient and an elevational gradient in Northeast China. In order to build the mathematical models for simulating the relationships of compositional and structural attributes of the Pinus koraiensis mixed forest with climatic and non-climatic factors, stepwise linear regression analyses were performed, incorporating 14 dependent variables and the linear and quadratic components of 9 factors. All the selected new models were computed under the +2°C and +10% precipitation and +4°C and +10% precipitation scenarios. The Max Temperature of Warmest Month, Mean Temperature of Warmest Quarter and Precipitation of Wettest Month were observed to be key climatic factors controlling the stand densities and total basal areas of Pinus koraiensis mixed forest. Increased summer temperatures and precipitations strongly enhanced the stand densities and total basal areas of broadleaf trees but had little effect on Pinus koraiensis under the +2°C and +10% precipitation scenario and +4°C and +10% precipitation scenario. CONCLUSIONS/SIGNIFICANCE: These results show that the Max Temperature of Warmest Month, Mean Temperature of Warmest Quarter and Precipitation of Wettest Month are key climatic factors which shape the composition and structure of Pinus koraiensis mixed forest. Although the Pinus koraiensis would persist, the current forests dominated by Pinus koraiensis in the region would all shift and become broadleaf-dominated forests due to the dramatic increase of broadleaf trees under the future global warming and increased precipitation.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Bioinformatic and phylogenetic analysis of the CLAVATA3/EMBRYO-SURROUNDING REGION (CLE) and the CLE-LIKE signal peptide genes in the Pinophyta.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1471-2229-14-47">10.1186/1471-2229-14-47</a></span>, <span>BMC plant biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>BACKGROUND: There is a rapidly growing awareness that plant peptide signalling molecules are numerous and varied and they are known to play fundamental roles in angiosperm plant growth and development. Two closely related peptide signalling molecule families are the CLAVATA3-EMBRYO-SURROUNDING REGION (CLE) and CLE-LIKE (CLEL) genes, which encode precursors of secreted peptide ligands that have roles in meristem maintenance and root gravitropism. Progress in peptide signalling molecule research in gymnosperms has lagged behind that of angiosperms. We therefore sought to identify CLE and CLEL genes in gymnosperms and conduct a comparative analysis of these gene families with angiosperms. RESULTS: We undertook a meta-analysis of the GenBank/EMBL/DDBJ gymnosperm EST database and the Picea abies and P. glauca genomes and identified 93 putative CLE genes and 11 CLEL genes among eight Pinophyta species, in the genera Cryptomeria, Pinus and Picea. The predicted conifer CLE and CLEL protein sequences had close phylogenetic relationships with their homologues in Arabidopsis. Notably, perfect conservation of the active CLE dodecapeptide in presumed orthologues of the Arabidopsis CLE41/44-TRACHEARY ELEMENT DIFFERENTIATION (TDIF) protein, an inhibitor of tracheary element (xylem) differentiation, was seen in all eight conifer species. We cloned the Pinus radiata CLE41/44-TDIF orthologues. These genes were preferentially expressed in phloem in planta as expected, but unexpectedly, also in differentiating tracheary element (TE) cultures. Surprisingly, transcript abundances of these TE differentiation-inhibitors sharply increased during early TE differentiation, suggesting that some cells differentiate into phloem cells in addition to TEs in these cultures. Applied CLE13 and CLE41/44 peptides inhibited root elongation in Pinus radiata seedlings. We show evidence that two CLEL genes are alternatively spliced via 3'-terminal acceptor exons encoding separate CLEL peptides. CONCLUSIONS: The CLE and CLEL genes are found in conifers and they exhibit at least as much sequence diversity in these species as they do in other plant species. Only one CLE peptide sequence has been 100% conserved between gymnosperms and angiosperms over 300 million years of evolutionary history, the CLE41/44-TDIF peptide and its likely conifer orthologues. The preferential expression of these vascular development-regulating genes in phloem in conifers, as they are in dicot species, suggests close parallels in the regulation of secondary growth and wood formation in gymnosperm and dicot plants. Based on our bioinformatic analysis, we predict a novel mechanism of regulation of the expression of several conifer CLEL peptides, via alternative splicing resulting in the selection of alternative C-terminal exons encoding separate CLEL peptides.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Radial stem growth in response to microclimate and soil moisture in a drought-prone mixed coniferous forest at an inner Alpine site.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1007/s10342-013-0777-z">10.1007/s10342-013-0777-z</a></span>, <span>European journal of forest research</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Dendroclimatological studies in a dry inner Alpine environment (750 m a.s.l.) revealed different growth response of co-occurring coniferous species to climate, which is assumed to be caused by a temporal shift in wood formation among species. The main focus of this study therefore was to monitor intra-annual dynamics of radial increment growth of mature deciduous and evergreen coniferous species (Pinus sylvestris, Larix decidua and Picea abies) during two consecutive years with contrasting climatic conditions. Radial stem growth was continuously followed by band dendrometers and modelled using Gompertz functions to determine time of maximum growth. Histological analyses of tree ring formation allowed determination of temporal dynamics of cambial activity and xylem cell development. Daily fluctuations in stem radius and radial stem increments were extracted from dendrometer traces, and correlations with environmental variables were performed. While a shift in temporal dynamics of radial growth onset and cessation was detected among co-occurring species, intra-annual radial growth peaked synchronously in late May 2011 and early June 2012. Moist atmospheric conditions, i.e. high relative air humidity, low vapour pressure deficit and low air temperature during the main growing period, favoured radial stem increment of all species. Soil water content and soil temperature were not significantly related to radial growth. Although a temporal shift in onset and cessation of wood formation was detected among species, synchronous culmination of radial growth indicates homogenous exogenous and/or endogenous control. The close coupling of radial growth to atmospheric conditions points to the importance of stem water status for intra-annual growth of drought-prone conifers.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/gb-2014-15-3-r59">10.1186/gb-2014-15-3-r59</a></span>, <span>Genome biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The size and complexity of conifer genomes has, until now, prevented full genome sequencing and assembly. The large research community and economic importance of loblolly pine, Pinus taeda L., made it an early candidate for reference sequence determination.We develop a novel strategy to sequence the genome of loblolly pine that combines unique aspects of pine reproductive biology and genome assembly methodology. We use a whole genome shotgun approach relying primarily on next generation sequence generated from a single haploid seed megagametophyte from a loblolly pine tree, 20-1010, that has been used in industrial forest tree breeding. The resulting sequence and assembly was used to generate a draft genome spanning 23.2 Gbp and containing 20.1 Gbp with an N50 scaffold size of 66.9 kbp, making it a significant improvement over available conifer genomes. The long scaffold lengths allow the annotation of 50,172 gene models with intron lengths averaging over 2.7 kbp and sometimes exceeding 100 kbp in length. Analysis of orthologous gene sets identifies gene families that may be unique to conifers. We further characterize and expand the existing repeat library based on the de novo analysis of the repetitive content, estimated to encompass 82% of the genome.In addition to its value as a resource for researchers and breeders, the loblolly pine genome sequence and assembly reported here demonstrates a novel approach to sequencing the large and complex genomes of this important group of plants that can now be widely applied.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Timber! Felling the loblolly pine genome.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/gb4170">10.1186/gb4170</a></span>, <span>Genome biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Conventional short read sequences derived from haploid DNA were extended into long super-reads enabling assembly of the massive 22 Gbp loblolly pine, Pinus taeda, genome.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Present-day vegetation helps quantifying past land cover in selected regions of the Czech Republic.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0100117">10.1371/journal.pone.0100117</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>The REVEALS model is a tool for recalculating pollen data into vegetation abundances on a regional scale. We explored the general effect of selected parameters by performing simulations and ascertained the best model setting for the Czech Republic using the shallowest samples from 120 fossil sites and data on actual regional vegetation (60 km radius). Vegetation proportions of 17 taxa were obtained by combining the CORINE Land Cover map with forest inventories, agricultural statistics and habitat mapping data. Our simulation shows that changing the site radius for all taxa substantially affects REVEALS estimates of taxa with heavy or light pollen grains. Decreasing the site radius has a similar effect as increasing the wind speed parameter. However, adjusting the site radius to 1 m for local taxa only (even taxa with light pollen) yields lower, more correct estimates despite their high pollen signal. Increasing the background radius does not affect the estimates significantly. Our comparison of estimates with actual vegetation in seven regions shows that the most accurate relative pollen productivity estimates (PPEs) come from Central Europe and Southern Sweden. The initial simulation and pollen data yielded unrealistic estimates for Abies under the default setting of the wind speed parameter (3 m/s). We therefore propose the setting of 4 m/s, which corresponds to the spring average in most regions of the Czech Republic studied. Ad hoc adjustment of PPEs with this setting improves the match 3-4-fold. We consider these values (apart from four exceptions) to be appropriate, because they are within the ranges of standard errors, so they are related to original PPEs. Setting a 1 m radius for local taxa (Alnus, Salix, Poaceae) significantly improves the match between estimates and actual vegetation. However, further adjustments to PPEs exceed the ranges of original values, so their relevance is uncertain.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Transcriptome analysis of callus from Picea balfouriana.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1471-2164-15-553">10.1186/1471-2164-15-553</a></span>, <span>BMC genomics</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>BACKGROUND: Picea likiangensis var. balfouriana (Rehd. et Wils.) Hillier ex Slavin (also known as Picea balfouriana) is an ecologically and economically important conifer that grows rapidly under optimum conditions and produces high-quality wood. It has a wide geographic distribution and is prevalent in southwest and eastern regions of China. Under suboptimal conditions, P. balfouriana grows slowly, which restricts its cultivation. Somatic embryogenesis has been used in the mass propagation of commercial species. However, low initiation rates are a common problem and the mechanisms involved in the induction of somatic embryogenesis are not fully understood. To understand the molecular mechanisms regulating somatic embryogenesis in P. balfouriana, high-throughput RNA-seq technology was used to investigate the transcriptomes of embryogenic and non-embryogenic tissues from three P. balfouriana genotypes. We compared the genes expressed in these tissues to identify molecular markers with embryogenic potential. RESULTS: A total of 55,078,846 nucleotide sequence reads were obtained for the embryogenic and non-embryogenic tissues of P. balfouriana, and 49.56% of them uniquely matched 22,295 (84.3%) of the 26,437 genes in the Picea abies genome database (Nature 497: 579-584, 2013). Differential gene expression analysis identified 1,418 differentially expressed genes (false discovery rate &lt;0.0001; fold change ≥2) in the embryogenic tissues relative to the non-embryogenic tissues, including 431 significantly upregulated and 987 significantly downregulated genes. KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway analysis revealed that the most significantly altered genes were involved in plant hormone signal transduction, metabolic pathways (starch and sucrose metabolism), and phenylalanine metabolism. CONCLUSIONS: We found that the initiation of embryogenic tissues affected gene expression in many KEGG pathways, but predominantly in plant hormone signal transduction, plant-pathogen interaction, and starch and sucrose metabolism. The changes in multiple pathways related to induction in the P. balfouriana embryogenic tissues described here, will contribute to a more comprehensive understanding of the mechanisms involved in the initiation of somatic embryogenesis. Additionally, we found that somatic embryogenesis receptor kinase (SERK), arabinogalactan proteins, and members of the WUS-related homeobox protein family may play important roles and could act as molecular markers in the early stage of somatic embryogenesis, as reported previously.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Characterization of microsatellite loci in lichen-forming fungi of Bryoria section Implexae (Parmeliaceae).</h1>
      <section>
        <p><span><a href="https://doi.org/undefined">undefined</a></span>, <span>Applications in plant sciences</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>•The locally rare, haploid, lichen-forming fungi Bryoria capillaris, B. fuscescens, and B. implexa are associated with boreal forests and belong to Bryoria sect. Implexae. Recent phylogenetic studies consider them to be conspecific. Microsatellite loci were developed to study population structure in Bryoria sect. Implexae and its response to ecosystem disturbances. •We developed 18 polymorphic microsatellite markers using 454 pyrosequencing data assessed in 82 individuals. The number of alleles per locus ranged from two to 13 with an average of 4.6. Nei's unbiased gene diversity, averaged over loci, ranged from 0.38 to 0.52. The markers amplified with all three species, except for markers Bi05, Bi15, and Bi18. •The new markers will allow the study of population subdivision, levels of gene introgression, and levels of clonal spread of Bryoria sect. Implexae. They will also facilitate an understanding of the effects of forest disturbance on genetic diversity of these lichen species.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A Picea abies linkage map based on SNP markers identifies QTLs for four aspects of resistance to Heterobasidion parviporum infection.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0101049">10.1371/journal.pone.0101049</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>A consensus linkage map of Picea abies, an economically important conifer, was constructed based on the segregation of 686 SNP markers in a F1 progeny population consisting of 247 individuals. The total length of 1889.2 cM covered 96.5% of the estimated genome length and comprised 12 large linkage groups, corresponding to the number of haploid P. abies chromosomes. The sizes of the groups (from 5.9 to 9.9% of the total map length) correlated well with previous estimates of chromosome sizes (from 5.8 to 10.8% of total genome size). Any locus in the genome has a 97% probability to be within 10 cM from a mapped marker, which makes the map suited for QTL mapping. Infecting the progeny trees with the root rot pathogen Heterobasidion parviporum allowed for mapping of four different resistance traits: lesion length at the inoculation site, fungal spread within the sapwood, exclusion of the pathogen from the host after initial infection, and ability to prevent the infection from establishing at all. These four traits were associated with two, four, four and three QTL regions respectively of which none overlapped between the traits. Each QTL explained between 4.6 and 10.1% of the respective traits phenotypic variation. Although the QTL regions contain many more genes than the ones represented by the SNP markers, at least four markers within the confidence intervals originated from genes with known function in conifer defence; a leucoanthocyanidine reductase, which has previously been shown to upregulate during H. parviporum infection, and three intermediates of the lignification process; a hydroxycinnamoyl CoA shikimate/quinate hydroxycinnamoyltransferase, a 4-coumarate CoA ligase, and a R2R3-MYB transcription factor.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Annotation and re-sequencing of genes from de novo transcriptome assembly of Abies alba (Pinaceae).</h1>
      <section>
        <p><span><a href="https://doi.org/undefined">undefined</a></span>, <span>Applications in plant sciences</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>•We present a protocol for the annotation of transcriptome sequence data and the identification of candidate genes therein using the example of the nonmodel conifer Abies alba. •A normalized cDNA library was built from an A. alba seedling. The sequencing on a 454 platform yielded more than 1.5 million reads that were de novo assembled into 25149 contigs. Two complementary approaches were applied to annotate gene fragments that code for (1) well-known proteins and (2) proteins that are potentially adaptively relevant. Primer development and testing yielded 88 amplicons that could successfully be resequenced from genomic DNA. •The annotation workflow offers an efficient way to identify potential adaptively relevant genes from the large quantity of transcriptome sequence data. The primer set presented should be prioritized for single-nucleotide polymorphism detection in adaptively relevant genes in A. alba.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/1471-2229-14-95">10.1186/1471-2229-14-95</a></span>, <span>BMC plant biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>BACKGROUND: A positive relationship between genome size and intron length is observed across eukaryotes including Angiosperms plants, indicating a co-evolution of genome size and gene structure. Conifers have very large genomes and longer introns on average than most plants, but impacts of their large genome and longer introns on gene structure has not be described. RESULTS: Gene structure was analyzed for 35 genes of Picea glauca obtained from BAC sequencing and genome assembly, including comparisons with A. thaliana, P. trichocarpa and Z. mays. We aimed to develop an understanding of impact of long introns on the structure of individual genes. The number and length of exons was well conserved among the species compared but on average, P. glauca introns were longer and genes had four times more intronic sequence than Arabidopsis, and 2 times more than poplar and maize. However, pairwise comparisons of individual genes gave variable results and not all contrasts were statistically significant. Genes generally accumulated one or a few longer introns in species with larger genomes but the position of long introns was variable between plant lineages. In P. glauca, highly expressed genes generally had more intronic sequence than tissue preferential genes. Comparisons with the Pinus taeda BACs and genome scaffolds showed a high conservation for position of long introns and for sequence of short introns. A survey of 1836 P. glauca genes obtained by sequence capture mostly containing introns &lt;1 Kbp showed that repeated sequences were 10× more abundant in introns than in exons. CONCLUSION: Conifers have large amounts of intronic sequence per gene for seed plants due to the presence of few long introns and repetitive element sequences are ubiquitous in their introns. Results indicate a complex landscape of intron sizes and distribution across taxa and between genes with different expression profiles.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Xylem and phloem phenology in co-occurring conifers exposed to drought.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1007/s00468-014-1026-x">10.1007/s00468-014-1026-x</a></span>, <span>Trees (Berlin, Germany : West)</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Variability in xylem and phloem phenology among years and species is caused by contrasting temperatures prevailing at the start of the growing season and species-specific sensitivity to drought.The focus of this study was to determine temporal dynamics of xylem and phloem formation in co-occurring deciduous and evergreen coniferous species in a dry inner Alpine environment (750&nbsp;m a.s.l., Tyrol, Austria). By repeated micro-sampling of the stem, timing of key phenological dates of xylem and phloem formation was compared among mature Pinus sylvestris, Larix decidua and Picea abies during two consecutive years. Xylem formation in P. sylvestris started in mid and late April 2011 and 2012, respectively, and in both years about 2 week later in P. abies and L. decidua. Phloem formation preceded xylem formation on average by 3 week in P. sylvestris, and c. 5 week in P. abies and L. decidua. Based on modeled cell number increase, tracheid production peaked between early through late May 2011 and late May through mid-June 2012. Phloem formation culminated between late April and mid-May in 2011 and in late May 2012. Production of xylem and phloem cells continued for about 4 and 5-6&nbsp;months, respectively. High variability in xylem increment among years and species is related to exogenous control by climatic factors and species-specific sensitivity to drought, respectively. On the other hand, production of phloem cells was quite homogenous and showed asymptotic decrease with respect to xylem cells indicating endogenous control. Results indicate that onset and culmination of xylem and phloem formation are controlled by early spring temperature, whereby strikingly advanced production of phloem compared to xylem cells suggests lower temperature requirement for initiation of the former.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Development and characterization of 25 EST-SSR markers in Pinus sylvestris var. mongolica (Pinaceae).</h1>
      <section>
        <p><span><a href="https://doi.org/undefined">undefined</a></span>, <span>Applications in plant sciences</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>•A set of novel expressed sequence tag (EST) microsatellite markers was developed in Pinus sylvestris var. mongolica to promote further genetic studies in this species. •One hundred seventy-five EST-simple sequence repeat (SSR) primers were designed and synthesized for 31,653 isotigs based on P. tabuliformis EST sequences. The primer pairs were used to identify 25 polymorphic loci in 48 individuals. The number of alleles ranged from two to eight with observed and expected heterozygosity values of 0.0435 to 0.8125 and 0.0430 to 0.7820, respectively. •These new polymorphic EST-SSR markers will be useful for assessing genetic diversity, molecular breeding and genetic improvement, and conservation of P. sylvestris var. mongolica.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Draft Genome Sequence of Burkholderia sordidicola S170, a Potential Plant Growth Promoter Isolated from Coniferous Forest Soil in the Czech Republic.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1128/genomea.00810-14">10.1128/genomea.00810-14</a></span>, <span>Genome announcements</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Burkholderia species are key players in the accumulation of carbon from cellulose decomposition in coniferous forest ecosystems. We report here the draft genome of Burkholderia sordidicola strain S170, containing features associated with known genes involved in plant growth promotion, the biological control of plant diseases, and green remediation technologies.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Highly informative single-copy nuclear microsatellite DNA markers developed using an AFLP-SSR approach in black spruce (Picea mariana) and red spruce (P. rubens).</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0103789">10.1371/journal.pone.0103789</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Microsatellites or simple sequence repeats (SSRs) are highly informative molecular markers for various biological studies in plants. In spruce (Picea) and other conifers, the development of single-copy polymorphic genomic microsatellite markers is quite difficult, owing primarily to the large genome size and predominance of repetitive DNA sequences throughout the genome. We have developed highly informative single-locus genomic microsatellite markers in black spruce (Picea mariana) and red spruce (Picea rubens) using a simple but efficient method based on a combination of AFLP and microsatellite technologies.A microsatellite-enriched library was constructed from genomic AFLP DNA fragments of black spruce. Sequencing of the 108 putative SSR-containing clones provided 94 unique sequences with microsatellites. Twenty-two of the designed 34 primer pairs yielded scorable amplicons, with single-locus patterns. Fourteen of these microsatellite markers were characterized in 30 black spruce and 30 red spruce individuals drawn from many populations. The number of alleles at a polymorphic locus ranged from 2 to 18, with a mean of 9.3 in black spruce, and from 3 to 15, with a mean of 6.2 alleles in red spruce. The polymorphic information content or expected heterozygosity ranged from 0.340 to 0.909 (mean = 0.67) in black spruce and from 0.161 to 0.851 (mean = 0.62) in red spruce. Ten SSR markers showing inter-parental polymorphism inherited in a single-locus Mendelian mode, with two cases of distorted segregation. Primer pairs for almost all polymorphic SSR loci resolved microsatellites of comparable size in Picea glauca, P. engelmannii, P. sitchensis, and P. abies.The AFLP-based microsatellite-enriched library appears to be a rapid, cost-effective approach for isolating and developing single-locus informative genomic microsatellite markers in black spruce. The markers developed should be useful in black spruce, red spruce and other Picea species for various genetics, genomics, breeding, forensics, conservation studies and applications.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Uniform versus asymmetric shading mediates crown recession in conifers.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0104187">10.1371/journal.pone.0104187</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>In this study we explore the impact of asymmetrical vs. uniform crown shading on the mortality and growth of upper and lower branches within tree crowns, for two conifer species: shade intolerant lodgepole pine (Pinus contorta) and shade tolerant white spruce (Picea glauca). We also explore xylem hydraulics, foliar nutrition, and carbohydrate status as drivers for growth and expansion of the lower and upper branches in various types of shading. This study was conducted over a two-year period across 10 regenerating forest sites dominated by lodgepole pine and white spruce, in the lower foothills of Alberta, Canada. Trees were assigned to one of four shading treatments: (1), complete uniform shading of the entire tree, (2) light asymmetric shading where the lower 1/4-1/3 of the tree crown was shaded, (3) heavy asymmetric shading as in (2) except with greater light reduction and (4) control in which no artificial shading occurred and most of the entire crown was exposed to full light. Asymmetrical shading of only the lower crown had a larger negative impact on the bud expansion and growth than did uniform shading, and the effect was stronger in pine relative to spruce. In addition, lower branches in pine also had lower carbon reserves, and reduced xylem-area specific conductivity compared to spruce. For both species, but particularly the pine, the needles of lower branches tended to store less C than upper branches in the asymmetric shade, which could suggest a movement of reserves away from the lower branches. The implications of these findings correspond with the inherent shade tolerance and self-pruning behavior of these conifers and supports a carbon based mechanism for branch mortality--mediated by an asymmetry in light exposure of the crown.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Entoloma subgenus Leptonia in boreal-temperate Eurasia: towards a phylogenetic species concept.</h1>
      <section>
        <p><span><a href="https://doi.org/10.3767/003158514x681774">10.3767/003158514x681774</a></span>, <span>Persoonia</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>This study reveals the concordance, or lack thereof, between morphological and phylogenetic species concepts within Entoloma subg. Leptonia in boreal-temperate Eurasia, combining a critical morphological examination with a multigene phylogeny based on nrITS, nrLSU and mtSSU sequences. A total of 16 taxa was investigated. Emended concepts of subg. Leptonia and sect. Leptonia as well as the new sect. Dichroi are presented. Two species (Entoloma percoelestinum and E. sublaevisporum) and one variety (E. tjallingiorum var. laricinum) are described as new to science. On the basis of the morphological and phylogenetical evidence E. alnetorum is reduced to a variety of E. tjallingiorum, and E. venustum is considered a variety of E. callichroum. Accordingly, the new combinations E. tjallingiorum var. alnetorum and E. callichroum var. venustum are proposed. Entoloma lepidissimum var. pauciangulatum is now treated as a synonym of E. chytrophilum. Neotypes for E. dichroum, E. euchroum and E. lampropus are designated.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Comparative in silico analysis of EST-SSRs in angiosperm and gymnosperm tree genera.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1186/s12870-014-0220-8">10.1186/s12870-014-0220-8</a></span>, <span>BMC plant biology</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>BACKGROUND: Simple Sequence Repeats (SSRs) derived from Expressed Sequence Tags (ESTs) belong to the expressed fraction of the genome and are important for gene regulation, recombination, DNA replication, cell cycle and mismatch repair. Here, we present a comparative analysis of the SSR motif distribution in the 5'UTR, ORF and 3'UTR fractions of ESTs across selected genera of woody trees representing gymnosperms (17 species from seven genera) and angiosperms (40 species from eight genera). RESULTS: Our analysis supports a modest contribution of EST-SSR length to genome size in gymnosperms, while EST-SSR density was not associated with genome size in neither angiosperms nor gymnosperms. Multiple factors seem to have contributed to the lower abundance of EST-SSRs in gymnosperms that has resulted in a non-linear relationship with genome size diversity. The AG/CT motif was found to be the most abundant in SSRs of both angiosperms and gymnosperms, with a relative increase in AT/AT in the latter. Our data also reveals a higher abundance of hexamers across the gymnosperm genera. CONCLUSIONS: Our analysis provides the foundation for future comparative studies at the species level to unravel the evolutionary processes that control the SSR genesis and divergence between angiosperm and gymnosperm tree species.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A new tool for exploring climate change induced range shifts of conifer species in China.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0098643">10.1371/journal.pone.0098643</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>It is inevitable that tree species will undergo considerable range shifts in response to anthropogenic induced climate change, even in the near future. Species Distribution Models (SDMs) are valuable tools in exploring general temporal trends and spatial patterns of potential range shifts. Understanding projections to future climate for tree species will facilitate policy making in forestry. Comparative studies for a large number of tree species require the availability of suitable and standardized indices. A crucial limitation when deriving such indices is the threshold problem in defining ranges, which has made interspecies comparison problematic until now. Here we propose a set of threshold-free indices, which measure range explosion (I), overlapping (O), and range center movement in three dimensions (Dx, Dy, Dz), based on fuzzy set theory (Fuzzy Set based Potential Range Shift Index, F-PRS Index). A graphical tool (PRS_Chart) was developed to visualize these indices. This technique was then applied to 46 Pinaceae species that are widely distributed and partly common in China. The spatial patterns of the modeling results were then statistically tested for significance. Results showed that range overlap was generally low; no trends in range size changes and longitudinal movements could be found, but northward and poleward movement trends were highly significant. Although range shifts seemed to exhibit huge interspecies variation, they were very consistent for certain climate change scenarios. Comparing the IPCC scenarios, we found that scenario A1B would lead to a larger extent of range shifts (less overlapping and more latitudinal movement) than the A2 and the B1 scenarios. It is expected that the newly developed standardized indices and the respective graphical tool will facilitate studies on PRS's for other tree species groups that are important in forestry as well, and thus support climate adaptive forest management.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Climatic influences on wood anatomy and tree-ring features of Great Basin conifers at a new mountain observatory.</h1>
      <section>
        <p><span><a href="https://doi.org/undefined">undefined</a></span>, <span>Applications in plant sciences</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>•A network of mountain observing stations has been installed in the Great Basin of North America. NevCAN (Nevada Climate-ecohydrological Assessment Network), which spans a latitudinal range of 2.5° and two elevation ranges of about 2000 m each, enabled us to investigate tree growth in relation to climate. •We analyzed wood anatomy and tree-ring characteristics of four conifer species in response to different levels of water availability by comparing a low- and a high-elevation population. Chronologies of earlywood and latewood widths, as well as cellular parameters, were developed from the year 2000 to 2012. •At the southern (drier and warmer) sites, Pinus monophylla had smaller cell lumen, tracheid diameter, and cell wall thickness. Pinus monophylla and P. flexilis showed bigger cellular elements at the higher elevations, whereas the opposite pattern was found in Picea engelmannii and Pinus longaeva. When all species and sites were pooled together, stem diameter was positively related with earlywood anatomical parameters. •We have provided a glimpse of the applications that NevCAN, as a new scientific tool, could allow in the general field of botany. In particular, we were able to investigate how differences in water stress related to elevation lead to changes in xylem anatomy.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Identification and evaluation of strain B37 of Bacillus subtilis antagonistic to sapstain fungi on poplar wood.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1155/2014/149342">10.1155/2014/149342</a></span>, <span>TheScientificWorldJournal</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>Devaluation of poplar products by sapstain accounts for huge and unpredictable losses each year in China. We had isolated four poplar sapstain fungi, Ceratocystis adiposa Hz91, Lasiodiplodia theobromae YM0737, L. theobromae Fx46, and Fusarium sp. YM05, from five poplar varieties and 13 antagonistic bacteria from nine diverse varieties. After being experimented with agar plates, wood chips, and enzyme activities, strain B37 was identified as the best poplar sapstain biocontrol bacterium. The strain B37 was identified as Bacillus subtilis using sequences of the 16S rRNA gene, physiological biochemical, and morphological characteristics.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A molecular identification protocol for roots of boreal forest tree species.</h1>
      <section>
        <p><span><a href="https://doi.org/undefined">undefined</a></span>, <span>Applications in plant sciences</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>•Roots play a key role in many ecological processes, yet our ability to identify species from bulk root samples is limited. Molecular tools may be used to identify species from root samples, but they have not yet been developed for most systems. Here we present a PCR-based method previously used to identify roots of grassland species, modified for use in boreal forests. •We used repeatable interspecific size differences in fluorescent amplified fragment length polymorphisms of three noncoding chloroplast DNA regions to identify seven woody species common to boreal forests in Alberta, Canada. •Abies balsamea, Alnus crispa, Betula papyrifera, Pinus contorta, and Populus tremuloides were identifiable to species, while Picea glauca and Picea mariana were identifiable to genus. In mixtures of known composition of foliar DNA, species were identified with 98% accuracy using one region. Mixed root samples of unknown composition were identified with 100% accuracy; four species were identified using one region, while three species were identified using two regions. •This methodology is accurate, efficient, and inexpensive, and thus a valuable approach for ecological studies of roots. Furthermore, this method has now been validated for both grassland and boreal forest systems, and thus may also have applications in any plant community.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>The relationship between species diversity and genetic structure in the rare Picea chihuahuana tree species community, Mexico.</h1>
      <section>
        <p><span><a href="https://doi.org/10.1371/journal.pone.0111623">10.1371/journal.pone.0111623</a></span>, <span>PloS one</span></p>
      </section>
    </header>
    <img src="https://plos.org/images/favicon.ico">
    <main>Species diversity and genetic diversity, the most basic elements of biodiversity, have long been treated as separate topics, although populations evolve within a community context. Recent studies on community genetics and ecology have suggested that genetic diversity is not completely independent of species diversity. The Mexican Picea chihuahuana Martínez is an endemic species listed as "Endangered" on the Red List. Forty populations of Chihuahua spruce have been identified. This species is often associated with tree species of eight genera in gallery forests. This rare Picea chihuahuana tree community covers an area no more than 300 ha and has been subject of several studies involving different topics such as ecology, genetic structure and climate change. The overall aim of these studies was to obtain a dataset for developing management tools to help decision makers implement preservation and conservation strategies. However, this unique forest tree community may also represent an excellent subject for helping us to understand the interplay between ecological and evolutionary processes in determining community structure and dynamics. The AFLP technique and species composition data were used together to test the hypothesis that species diversity is related to the adaptive genetic structure of some dominant tree species (Picea chihuahuana, Pinus strobiformis, Pseudotsuga menziesii and Populus tremuloides) of the Picea chihuahuana tree community at fourteen locations. The Hill numbers were used as a diversity measure. The results revealed a significant correlation between tree species diversity and genetic structure in Populus tremuloides. Because the relationship between the two levels of diversity was found to be positive for the putative adaptive AFLP detected, genetic and species structures of the tree community were possibly simultaneously adapted to a combination of ecological or environmental factors. The present findings indicate that interactions between genetic variants and species diversity may be crucial in shaping tree communities.<span class="show">...</span><span class="hide"></span></main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A taxonomic monograph of Nearctic Scolytus Geoffroy (Coleoptera, Curculionidae, Scolytinae).</h1>
      <section>
        <p><span><a href="https://doi.org/10.3897/zookeys.450.7452">10.3897/zookeys.450.7452</a></span>, <span>ZooKeys</span></p>
      </section>
    </header>
    <img src="undefined">
    <main>The Nearctic bark beetle genus Scolytus Geoffroy was revised based in part on a molecular and morphological phylogeny. Monophyly of the native species was tested using mitochondrial (COI) and nuclear (28S, CAD, ArgK) genes and 43 morphological characters in parsimony and Bayesian phylogenetic analyses. Parsimony analyses of molecular and combined datasets provided mixed results while Bayesian analysis recovered most nodes with posterior probabilities &gt;90%. Native hardwood- and conifer-feeding Scolytus species were recovered as paraphyletic. Native Nearctic species were recovered as paraphyletic with hardwood-feeding species sister to Palearctic hardwood-feeding species rather than to native conifer-feeding species. The Nearctic conifer-feeding species were monophyletic. Twenty-five species were recognized. Four new synonyms were discovered: Scolytuspraeceps LeConte, 1868 (= Scolytusabietis Blackman, 1934; = Scolytusopacus Blackman, 1934), Scolytusreflexus Blackman, 1934 (= Scolytusvirgatus Bright, 1972; = Scolytuswickhami Blackman, 1934). Two species were reinstated: Scolytusfiskei Blackman, 1934 and Scolytussilvaticus Bright, 1972. A diagnosis, description, distribution, host records and images were provided for each species and a key is presented to all species.<span class="show">...</span><span class="hide"></span></main>
  </div>
</article>
<div class="genera">
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4713104</span><span class="count">77</span></li>
        <li><span class="pmcid">PMC4389208</span><span class="count">24</span></li>
        <li><span class="pmcid">PMC4395334</span><span class="count">23</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC4931021</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4574200</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4619678</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4061133</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3914916</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4811566</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4543934</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3753317</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4725448</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4305089</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4873231</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4705065</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4702125</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4666671</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4943178</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4874406</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4794938</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4769139</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4723224</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4709368</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4700082</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4684699</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4669935</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4557068</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4495060</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4409050</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4351838</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312774</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Arabidopsis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4016512</span><span class="count">38</span></li>
        <li><span class="pmcid">PMC4919339</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4746482</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4619436</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4527274</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4867723</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3679494</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4621400</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4570457</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4523189</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4937839</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4573423</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4692408</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4414588</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4053751</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4938651</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4851370</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4757406</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4576002</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4528325</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Populus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4312940</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4221877</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4811566</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4938651</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4612159</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4222919</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pinus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4855863</span><span class="count">39</span></li>
        <li><span class="pmcid">PMC4950538</span><span class="count">36</span></li>
        <li><span class="pmcid">PMC4637882</span><span class="count">21</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">12</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">12</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4931021</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4574200</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4689564</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4543934</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4061133</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4811566</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4684699</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4401641</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4686996</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3837413</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4906659</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4705065</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4700082</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4691086</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4666671</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4181978</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3753317</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4943178</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4912770</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4909799</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4909774</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4893459</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4852653</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4846807</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4794938</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4769139</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4768284</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Taxus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4681332</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Fusarium</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4221877</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Heterobasidion</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4495060</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pinaceae</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4684699</span><span class="count">41</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4495060</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4758241</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4545727</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4409050</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Larix</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4931021</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4181978</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3997487</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4758241</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4702125</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630587</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4305089</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4249714</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3739742</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Abies</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4061133</span><span class="count">21</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4700082</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4388536</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4943178</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4702125</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4574200</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4564278</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4181978</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3997487</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4620457</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4874406</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4873231</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4617160</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4485965</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4329319</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3837413</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3739742</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Alnus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4061133</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4931021</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4612159</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4873231</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4716518</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4355648</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4305089</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3753317</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Quercus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4855863</span><span class="count">31</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4061133</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4612159</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3904049</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4811566</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4415794</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4305089</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3753317</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4827862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4758377</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Gymny</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Class</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Order</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Length</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">3</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Drosophila</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4860704</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4776372</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4717527</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Anopheles</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Betula</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4312940</span><span class="count">23</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4873231</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4947481</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630587</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4620457</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4557068</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4061133</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Salix</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4612159</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4061133</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4829932</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4357446</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4355648</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Aesculus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3904049</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Hymenaea</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Bursaphelenchus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3823978</span><span class="count">7</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Monochamus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3823978</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Schistonchus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3823978</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Seinura</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3823978</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Eucalyptus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4160553</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4811566</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4329319</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4138101</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Ginkgo</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3836751</span><span class="count">35</span></li>
        <li><span class="pmcid">PMC4160553</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4681332</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cycas</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4160553</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4758241</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Selaginella</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4681332</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4527274</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Gnetum</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4160553</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Gingko</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3836751</span><span class="count">5</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pityophthorus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3837413</span><span class="count">16</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Araptus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3837413</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cinara</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">34</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">30</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">25</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cupressobium</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3909620</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Buchnera</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pseudotsuga</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4222919</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cupressus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4811566</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4619678</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4222919</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Size</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Different</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Persistence</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Support</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Tsuga</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4943178</span><span class="count">22</span></li>
        <li><span class="pmcid">PMC4738281</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4415794</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cedrus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4574200</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4943178</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4764356</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4681332</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4181978</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3909620</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Megastigmus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Dendroctonus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4832482</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Dryas</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4523770</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Artemisia</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4357446</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC4543934</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Carpinus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4061133</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3885877</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4846807</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3904049</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
</div>
<div class="species">
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea abies</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4691297</span><span class="count">55</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">47</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">39</span></li>
        <li><span class="pmcid">PMC4263092</span><span class="count">39</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">36</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">36</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">34</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">29</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">27</span></li>
        <li><span class="pmcid">PMC4035765</span><span class="count">26</span></li>
        <li><span class="pmcid">PMC3914916</span><span class="count">23</span></li>
        <li><span class="pmcid">PMC4561357</span><span class="count">20</span></li>
        <li><span class="pmcid">PMC4545727</span><span class="count">19</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">17</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">16</span></li>
        <li><span class="pmcid">PMC4315866</span><span class="count">16</span></li>
        <li><span class="pmcid">PMC3706884</span><span class="count">15</span></li>
        <li><span class="pmcid">PMC4783019</span><span class="count">12</span></li>
        <li><span class="pmcid">PMC4564692</span><span class="count">12</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC3837413</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4351073</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4723141</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4094777</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4938651</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4801349</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4586145</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4495060</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3904049</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3679494</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4666671</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC3753317</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4527274</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4346200</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4908602</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4899478</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4894875</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4851370</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4740435</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4573423</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4249714</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4963455</span><span class="count">3</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea glauca</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4108047</span><span class="count">95</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">40</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">30</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">21</span></li>
        <li><span class="pmcid">PMC3837413</span><span class="count">20</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">20</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC4665433</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4388536</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4912770</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4927954</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4222544</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4681332</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4138101</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4938651</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4758241</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4739566</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4721073</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4619436</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4134192</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4947481</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4942103</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4908464</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4797215</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4777116</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4684699</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4570457</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4528325</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4523189</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4123385</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4053751</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea sitchensis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3543189</span><span class="count">19</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4691086</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4495060</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4263092</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4134192</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4909774</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4721073</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4499884</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4414588</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4053751</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pinus taeda</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4691297</span><span class="count">48</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">45</span></li>
        <li><span class="pmcid">PMC4528325</span><span class="count">42</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">29</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">24</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">21</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">21</span></li>
        <li><span class="pmcid">PMC4691086</span><span class="count">15</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4053751</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4909799</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4909774</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4723224</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4684699</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4053979</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4844950</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4769139</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4935725</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4919339</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4860704</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4857224</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4827862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4795606</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4757406</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4649691</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4627733</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4621400</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4617160</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4570457</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4529292</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4414588</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4351838</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312774</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4138101</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4105350</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Taxus mairei</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4681332</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Drosophila melanogaster</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4717527</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC4938651</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4867723</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A. thaliana</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4676809</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4570457</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4251312</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea taeda</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4108047</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4221877</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea menziesii</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4388536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Arabidopsis thaliana</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4302573</span><span class="count">30</span></li>
        <li><span class="pmcid">PMC4529292</span><span class="count">29</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">23</span></li>
        <li><span class="pmcid">PMC4659896</span><span class="count">22</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">16</span></li>
        <li><span class="pmcid">PMC4527274</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4851370</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4769139</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4867723</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4251312</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4094777</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4692408</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4053751</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4964460</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4937839</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4919339</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4758241</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4746482</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4691297</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4630563</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4621400</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4619436</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4570457</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4528325</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4523189</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4263092</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4053979</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3706884</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Populus trichocarpa</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4769139</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4529292</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4851370</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4251312</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3706761</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3679494</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea trichocarpa</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4108047</span><span class="count">17</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Arabidopsis lyrata</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4851370</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4607528</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4529292</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea tremula</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4014479</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>E. coli</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4757406</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4687524</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4251312</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3543189</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Heterobasidion annosum</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">24</span></li>
        <li><span class="pmcid">PMC4705065</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4788975</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4773076</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4676809</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Heterobasidion irregulare</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">14</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Heterobasidion occidentale</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">5</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Saccharomyces cerevisiae</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4730770</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4755536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pleurotus ostreatus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Gibberella zeae</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">2</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Nectria haematococca</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pinus sylvestris</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4919910</span><span class="count">50</span></li>
        <li><span class="pmcid">PMC4723141</span><span class="count">44</span></li>
        <li><span class="pmcid">PMC4346200</span><span class="count">35</span></li>
        <li><span class="pmcid">PMC4110670</span><span class="count">34</span></li>
        <li><span class="pmcid">PMC4951524</span><span class="count">27</span></li>
        <li><span class="pmcid">PMC4123385</span><span class="count">16</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC3837413</span><span class="count">14</span></li>
        <li><span class="pmcid">PMC4800674</span><span class="count">12</span></li>
        <li><span class="pmcid">PMC4893459</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4329319</span><span class="count">10</span></li>
        <li><span class="pmcid">PMC4909799</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4150075</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC3904049</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4817349</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4649691</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4355648</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4656839</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4561357</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4499884</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3997487</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4947481</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4912597</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4899467</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4852653</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4788975</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4686996</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4611751</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4351838</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4945685</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4942459</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4935725</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4929562</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4913902</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4908464</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4894875</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4860704</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4858752</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4849635</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4795606</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4740435</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4715454</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Coccidioides posadasii</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Coccidioides immitis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Saccharomyces paradoxus</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Neurospora crassa</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4221877</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Schizosaccharomyces pombe</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4755536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Cryptococcus neoformans</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pseudomonas syringae</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4851370</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Hyaloperonospora parasitica</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Candida albicans</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4730770</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4947481</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Ustilago maydis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Sclerotinia sclerotiorum</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Botrytis cinerea</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Magnaporthe oryzae</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Fusarium graminearum</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4963455</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Colletotrichum higginsianum</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Neisseria meningitidis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Coccidioides albicans</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3547014</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pinus contorta</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4723141</span><span class="count">64</span></li>
        <li><span class="pmcid">PMC4222544</span><span class="count">13</span></li>
        <li><span class="pmcid">PMC4138101</span><span class="count">7</span></li>
        <li><span class="pmcid">PMC4909774</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC4721073</span><span class="count">6</span></li>
        <li><span class="pmcid">PMC3679494</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4912597</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4676862</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4016512</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC3848992</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4755556</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4732764</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4718046</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4609829</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4401641</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312774</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4302573</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4251312</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4233402</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4014611</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3948814</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3836751</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea sylvestris</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4035765</span><span class="count">20</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">8</span></li>
        <li><span class="pmcid">PMC4935725</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4860704</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC3701499</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4942462</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4700082</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4108047</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea torreyana</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>A. pinsapo</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Populus tremula</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4312940</span><span class="count">27</span></li>
        <li><span class="pmcid">PMC3679494</span><span class="count">11</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC3930551</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4887491</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4523770</span><span class="count">4</span></li>
        <li><span class="pmcid">PMC4947481</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4744691</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4419791</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4788975</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4723141</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4612159</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4499884</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4014479</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Pseudotsuga menziesii</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4233402</span><span class="count">148</span></li>
        <li><span class="pmcid">PMC4713104</span><span class="count">26</span></li>
        <li><span class="pmcid">PMC4222919</span><span class="count">9</span></li>
        <li><span class="pmcid">PMC4312940</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC4612159</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4329319</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4945685</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4755556</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4942103</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4860704</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4777116</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4758377</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4740435</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4726009</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4723141</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4721073</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4718046</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4619678</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4401641</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4388536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4312774</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4189497</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4138101</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3914916</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3799627</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea contorta</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC3664019</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4388536</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Larix occidentalis</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4233402</span><span class="count">18</span></li>
        <li><span class="pmcid">PMC4780373</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea omorika</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4777116</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
  <div class="card" tabindex="-1">
    <header>
      <h1>Picea pinaster</h1>
    </header>
    <main>
      <ul>
        <li><span class="pmcid">PMC4942462</span><span class="count">5</span></li>
        <li><span class="pmcid">PMC3830773</span><span class="count">3</span></li>
        <li><span class="pmcid">PMC4929119</span><span class="count">2</span></li>
        <li><span class="pmcid">PMC4103950</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3762812</span><span class="count">1</span></li>
        <li><span class="pmcid">PMC3664019</span><span class="count">1</span></li>
      </ul>
    </main>
  </div>
</div>
  
</body></html>